packages/google_maps_flutter/google_maps_flutter_android/test/google_maps_flutter_android_test.dart

```diff
@@ -25,12 +25,16 @@ void main() {
     required int mapId,
     required Future<dynamic>? Function(MethodCall call) handler,
   }) {
-    maps
-        .ensureChannelInitialized(mapId)
-        .setMockMethodCallHandler((MethodCall methodCall) {
-      log.add(methodCall.method);
-      return handler(methodCall);
-    });
+    final MethodChannel channel = maps.ensureChannelInitialized(mapId);
+    _ambiguate(TestDefaultBinaryMessengerBinding.instance)!
+        .defaultBinaryMessenger
+        .setMockMethodCallHandler(
+      channel,
```

Comment:
Slight deviation of the pattern here, expected:

```dart
_ambiguate(TestDefaultBinaryMessengerBinding.instance)!
    .defaultBinaryMessenger
    .setMockMethodCallhandler(
  maps.ensureChannelInitialized(mapId), (MethodCall methodCall) {...
```

(I think it's better extracting the channel to its own local, tbh)

Reply:
weird, i wonder why that happened.

Reply:
oh, i think this was the file i ended up looking at first, before i automated it, so this was done by hand

Reply:
to clarify, did you want me to make the changes consistent, or should i just land it as-is?
(this code will all get refactored in a few months anyway when the _ambiguate is removed.)

Reply:
(i'm going to mark this autosubmit for now given the repo merge, happy to land a separate PR if you think it's worth making the call sites more consistent)

Reply:
@Hixie nah, I just left these comments in case others wanted to look at what was "different" from the more automatic/mechanic changes. Thanks for landing this!